ui/app/utils/default-job-templates.js

```diff
@@ -0,0 +1,61 @@
+import helloWorld from './default_jobs/hello-world';
+import parameterized from './default_jobs/parameterized';
+import serviceDiscovery from './default_jobs/service-discovery';
+import variables from './default_jobs/variables';
+
+export default [
+  {
+    id: 'nomad/job-templates/default/hello-world',
+    keyValues: [
+      {
+        key: 'template',
+        value: helloWorld,
+      },
+      {
+        key: 'description',
+        value: 'A simple job that runs a single task on a single node.',
+      },
+    ],
```

Comment:
`blocking`: We should get the schema for the `keyValues` right here because these default job templates will not be able to be deleted.

Reply:
This is our standard variable keyValue typedef; https://github.com/hashicorp/nomad/blob/bd82a512c5a334c06012aa4dfa6a7bddbfd22451/ui/app/models/variable.js#L10-L15